plugins/modules/monitoring/airbrake_deployment.py

```diff
@@ -16,14 +16,21 @@
 DOCUMENTATION = '''
 ---
 module: airbrake_deployment
-author: "Bruce Pennypacker (@bpennypacker)"
+author:
+- "Bruce Pennypacker (@bpennypacker)"
+- "Patrick Humpal (@phumpal)"
 short_description: Notify airbrake about app deployments
 description:
-   - Notify airbrake about app deployments (see http://help.airbrake.io/kb/api-2/deploy-tracking)
+   - Notify airbrake about app deployments (see https://airbrake.io/docs/api/#deploys-v4)
+   - Parameter 'token' is deprecated in 2.10. Please remove entry.
 options:
-  token:
+  project_id:
+    description:
+      - Airbrake PROJECT_ID
+    required: true
```

Comment:
This change breaks backwards compatibiliy. Existing tasks will suddenly stop working, since they are missing the new options `project_id` and `project_key`. You need to implement some fallback mechanism that keeps existing playbooks working.